available_services/irrigationservice.yml

```diff
@@ -0,0 +1,40 @@
+services:
+  irr_db:
+    image: postgres:16.3
+    environment:
+      POSTGRES_DB: ${IRR_POSTGRES_DB}
+      POSTGRES_USER: ${IRR_POSTGRES_USER}
+      POSTGRES_PASSWORD: ${IRR_POSTGRES_PASSWORD}
+      PGPORT: ${IRR_POSTGRES_PORT}
+    healthcheck:
+      test: pg_isready -U ${IRR_POSTGRES_USER} -d ${IRR_POSTGRES_DB}
+      start_period: 10s
+      retries: 3
+      interval: 10s
+  irr_backend:
```

Comment:
@prske could you change the service names to be irrigation and irrigation_db? or something on those lines, just to follow a closer naming convention to the other services.